hub/alembic/versions/3cd5c742583c_convert_message_sa_datetime_to_sa_.py

```diff
@@ -0,0 +1,37 @@
+"""Convert Message sa.DateTime to sa.Timestamp.
+
+Revision ID: 3cd5c742583c
+Revises: 26e1e353eb58
+Create Date: 2025-02-05 14:13:17.636820
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision: str = "3cd5c742583c"
+down_revision: Union[str, None] = "26e1e353eb58"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.add_column("messages", sa.Column("created_at_temp", mysql.TIMESTAMP(fsp=6), server_default=sa.func.now()))
```

Comment:
But `fsp=6` is the important bit, right?  Did we need to (or do we want to) move from `DateTime` to `Timestamp` too?